Source/CesiumRuntime/Private/CesiumEncodedMetadataConversions.cpp

```diff
@@ -133,7 +133,7 @@ template <typename T>
 void coerceAndEncodeArrays(
     const FCesiumPropertyTablePropertyDescription& propertyDescription,
     const FCesiumPropertyTableProperty& property,
-    gsl::span<std::byte>& textureData,
+    const gsl::span<std::byte>& textureData,
```

Comment:
Not a suggestion, I'm just surprised we can add `const` here considering that we're overriding the contents of `textureData` 😮

Reply:
The `const `keyword is denoting that the `gsl::span` object is not modifiable. Internally it's just keeping track of a a pointer and size.

If you additionally changed this to `const gsl::span<const std::byte>`, this would declare that the byte contents are constant, and you'll get the compile error you would have originally expected.

Reply:
Yeah what Brian said, but I'll also point out that the `gsl::span` behavior is different from `std::vector`. If you have a const vector, the things in it are treated as const as well. This maybe makes sense because a vector owns its contents while a span just points to something owned elsewhere.